test-utils/src/main/java/module-info.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+module io.ballerina.stdlib.http.testutils {
+    requires io.netty.buffer;
+    requires io.netty.codec.http;
+    requires io.netty.common;
+    requires org.slf4j;
+    requires org.testng;
+    requires io.ballerina.runtime;
+    requires io.netty.transport;
+    requires io.ballerina.stdlib.http;
+
```

Comment:
Lets remove this line

Reply:
Fixed via [026e41b](https://github.com/ballerina-platform/module-ballerina-http/pull/1354/commits/026e41b61cf4e0a745cf14b2903f388633a30a0d)